src/NoSelfTransformer.ts

```diff
@@ -0,0 +1,10 @@
+import * as ts from "typescript";
+
+const transformSourceFile: ts.Transformer<ts.SourceFile> = node => {
+    const empty = ts.createEmptyStatement();
```

Comment:
```suggestion
    const empty = ts.createNotEmittedStatement(undefined!);
```

(I've left transformer comment before not emitted statement one)